docs/extending-machetli.rst

```diff
@@ -124,7 +124,13 @@ generators can directly access entities like sections, included packages, etc.
 .. code-block:: python
     :linenos:
 
+    from pathlib import Path
+    from machetli import tools
+
     def generate_initial_state(path: Path):
+        initial_state_dir = tools.get_eval_dir() / "iteration_00000"
```

Comment:
I don't like that `iteration_00000` is hardcoded here and in the PDDL and SAS modules.